pagseguro.php

```diff
@@ -624,54 +594,54 @@ private function validateOrderMessage()
 
         return Configuration::updateValue('PAGSEGURO_MESSAGE_ORDER_ID', $orderMensagem->id);
     }
-    
-	private function uninstallOrderMessage()
-    {
-    	$orders = array();
-    	$sql = "SELECT `id_order_message` as id FROM `"._DB_PREFIX_."order_message_lang` WHERE `name` = 'cart recovery pagseguro'";
-    	$result = (Db::getInstance(_PS_USE_SQL_SLAVE_)->executeS($sql));
-    
-    	if ($result) {
-    
-    		$bool = false;
-    		foreach ($result as $order_message) {
-    
-    			if (!$bool) {
-    
-    				$orders[] = $order_message['id'];
-    				$bool = true;
-    			} else {
-    
-    				if ( array_search($order_message['id'], $orders) === false){
-    					$orders[] = $order_message['id'];
-    				}
-    			}
-    		}
-    
-    		for($i = 0; $i < count($orders) ;$i++){
-    
-    			$sql = "DELETE FROM `"._DB_PREFIX_."order_message` WHERE `id_order_message` = '".$orders[$i]."'";
-    			Db::getInstance(_PS_USE_SQL_SLAVE_)->execute($sql);
-    		}
-    
-    		for($i = 0; $i < count($result) ;$i++){
-    			$id = $result[$i]['id'];
-    			$sql = "DELETE FROM `"._DB_PREFIX_."order_message_lang` WHERE `id_order_message` = '".$id."'";
-    			Db::getInstance(_PS_USE_SQL_SLAVE_)->execute($sql);
-    		}
-    		return true;
-    	}
-    	return false;
-    }
-
-    private function generatePagSeguroOrderStatus()
-    {
+
+    private function uninstallOrderMessage() {
+
+        $orders = array();
+        $sql = "SELECT `id_order_message` as id FROM `"._DB_PREFIX_."order_message_lang` WHERE `name` = 'cart recovery pagseguro'";
+        $result = (Db::getInstance(_PS_USE_SQL_SLAVE_)->executeS($sql));
+
+        if ($result) {
+
+            $bool = false;
+            foreach ($result as $order_message) {
+
+                if (!$bool) {
+
+                    $orders[] = $order_message['id'];
+                    $bool = true;
+                } else {
+
+                    if ( array_search($order_message['id'], $orders) === false){
+                        $orders[] = $order_message['id'];
+                    }
+                }
+            }
+
+            for($i = 0; $i < count($orders) ;$i++){
+
+                $sql = "DELETE FROM `"._DB_PREFIX_."order_message` WHERE `id_order_message` = '".$orders[$i]."'";
```

Comment:
You should sanitize your data to prevent SQL injections. You should write ```pSQL($orders[$i])``` if you have a string here, or use the ```(int)``` cast for an integer.